packages/aeproject-config/package-lock.json

```diff
@@ -1,5 +1,12 @@
 {
   "name": "aeproject-config",
   "version": "2.2.0",
-  "lockfileVersion": 1
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
```

Comment:
Is this supposed to be an empty string?

Reply:
good question. this is the result I get when running `npm install` with npm version `7.12.1`

Reply:
@mradkov yep, it's about the new lockfile version -> https://nitayneeman.com/posts/catching-up-with-package-lockfile-changes-in-npm-v7/